commons/src/main/java/com/flipkart/flux/commons/dto/Eventsinfo.java

```diff
@@ -0,0 +1,31 @@
+package com.flipkart.flux.commons.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.Map;
+
+@NoArgsConstructor(access = AccessLevel.PACKAGE)
+@AllArgsConstructor
+public class Eventsinfo {
+
+    @Getter
+    @JsonProperty
+    private Map<String, EventData> eventData;
```

Comment:
1. Variable name must be plural right
2. Can you add doc around what the key of the map is